### PR TITLE
Adding tests to confirm that Targeting Idea and Traffic Estimator services are working correctly

### PR DESCRIPTION
**Pull request by davewalk**
The Paid Search project requires that Partner Gateway and by extension this repo supports the Adwords Targeting Idea and Traffic Estimator APIs. It turns out that the original gads implemented this, but we never confirmed that it worked. I've written sandbox tests to confirm that they do. It's worth noting that if you use a non-production account to run the tests, you won't get real results back, as I mentioned in my comments before the test.

I also added some instructions to the documentation to give some more insight into how to actually run the tests for this repo.

### CODE REVIEW

**Review by zubricky — APPROVED** [review]
I'd like to run this eventually, but I will approve for now.

**Review by ankush91 — APPROVED** [review]

**Comment by slofurno** [comment]
tests pass, lgtm

File: v201710/sandbox_test.go [hunk 1]
```diff
@@ -30,6 +30,181 @@ func getTestConfig() AuthConfig {
...
+	for _, idea := range ideas[0:5] {
+		fmt.Println(idea.TargetingIdea[0].Value)
+	}
+}
+
+func TestSandboxTargetingIdeaURLs(t *testing.T) {
+	config := getTestConfig()
+	srv := NewTargetingIdeaService(&config.Auth)
+
+	LIMIT := 100
```
Comment (chriscannon) [inline v201710/sandbox_test.go]: Do you want to set this as some global variable since you call it twice?

**Review by chriscannon — APPROVED** [review]

File: v201710/sandbox_test.go (hunk 1 above)
Comment (davewalk) [reply v201710/sandbox_test.go]: yep, updated

**Pull request merged**